nuget.config

```diff
@@ -2,6 +2,6 @@
 <configuration>
 	<packageSources>
 		<add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
-		<add key="MyGet" value="https://dotnet.myget.org/F/uwpcommunitytoolkit/api/v3/index.json" protocolVersion="3" />
+		<add key="AzureLatest" value="https://pkgs.dev.azure.com/dotnet/WindowsCommunityToolkit/_packaging/WindowsCommunityToolkit-MainLatest/nuget/v3/index.json" protocolVersion="3" />
 	</packageSources>
 </configuration>
```

Comment:
We should complete the #68 update, from the Toolkit repo:

```xml
<?xml version="1.0" encoding="utf-8"?>
<configuration>
  <packageSources>
    <clear />
    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
    <add key="AzureLatest" value="https://pkgs.dev.azure.com/dotnet/WindowsCommunityToolkit/_packaging/WindowsCommunityToolkit-MainLatest/nuget/v3/index.json" protocolVersion="3" />
  </packageSources>
  <disabledPackageSources>
    <clear />
  </disabledPackageSources>
</configuration>
```

Reply:
Do what exactly?

Reply:
I think Alex just fixed this with his PR.